### PR TITLE
VMware: Add new module : vmware_drs_rule_facts

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY
Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
 - New Module Pull Request

##### COMPONENT NAME
lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py
test/integration/targets/vmware_drs_rule_facts/aliases
test/integration/targets/vmware_drs_rule_facts/tasks/main.yml

##### ANSIBLE VERSION

```
2.5devel
```

### CODE REVIEW

**Comment by Akasurde** [comment]
@pdellaert @tchernomax Could you please take a look ?

File: lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+notes:
+- Tested on vSphere 6.5
+requirements:
+- python >= 2.6
+- PyVmomi
+options:
+  cluster_name:
+    description:
+    - Name of the cluster.
+    - DRS facts for then given cluster will be returned.
```
Comment (pdellaert) [inline lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py]: `then` -> `the`

File: lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    - This is required parameter if datacenter parameter is not provided.
+  datacenter:
+    description:
+    - Name of the datacenter.
+    - DRS facts for all cluster from given datacenter will be returned.
```
Comment (pdellaert) [inline lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py]: `clusters`

**Review by pdellaert — COMMENTED** [review]
Small typo's

File: lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py]: Done.

File: lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py (hunk 2 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_drs_rule_facts.py]: Done.

**Review by pdellaert — APPROVED** [review]
shipit

**Comment by Akasurde** [comment]
@pdellaert Thanks for review.

**Pull request merged**